sdk/iotcentral/Azure.ResourceManager.IotCentral/src/autorest.md

```diff
@@ -74,4 +74,10 @@ rename-rules:
   URI: Uri
   Etag: ETag|etag
 
+directive:
+  - from: iotcentral.json
+    where: $.definitions
+    transform: >
+      $.AppTemplate.properties.order['type'] = 'integer';
```

Comment:
Is this a swagger error?

Reply:
Yes, service team opened a pr to fix it https://github.com/Azure/azure-rest-api-specs/pull/19983